toolkit/tools/internal/timestamp/timestamp_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright Microsoft Corporation.
+// Licensed under the MIT License.
+
+package timestamp
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+)
+
+var (
+	info1 = New("tool 1", true)
+	info2 = New("tool 2", false)
+)
+
+//TestMain found in configuration_test.go.
+
+func Test_WritetoFile_range_instant(t *testing.T) {
```

Comment:
For unit tests, please follow the Arrange-Act-Assert (AAA) pattern.

Reply:
Deleting this file for this PR, will include a unit test file following the pattern in a new PR.